win32ss/drivers/videoprt/registry.c

```diff
@@ -578,14 +578,18 @@ IntCreateNewRegistryPath(
                                 OBJ_KERNEL_HANDLE | OBJ_CASE_INSENSITIVE,
                                 NULL,
                                 NULL);
-    Status = ZwOpenKey(&SettingsKey, KEY_READ, &ObjectAttributes);
+    Status = ZwOpenKey(&SettingsKey, KEY_READ | KEY_WRITE, &ObjectAttributes);
```

Comment:
That should work, though `KEY_QUERY_VALUE | KEY_WRITE` and `KEY_READ | KEY_SET_VALUE` should be enough and more expliclit.